Goal: The pam_smbpass examples are not in samples/ but examples/

Fixes: ?

Status wrt upstream: Debian specific

Note: 

Index: samba-3.0.25b/source/pam_smbpass/README
===================================================================
--- samba-3.0.25b.orig/source/pam_smbpass/README	2007-06-27 09:25:38.244112055 +0200
+++ samba-3.0.25b/source/pam_smbpass/README	2007-06-27 09:28:23.736267996 +0200
@@ -37,7 +37,7 @@
 	smbconf=<file>	-	specify an alternate path to the smb.conf
 				file.
 
-See the samples/ directory for example PAM configurations using this
+See the examples/ directory for example PAM configurations using this
 module.
 
 Thanks go to the following people:
